apps/ideas/serializers.py

```diff
@@ -86,6 +91,30 @@ def get_user_rating(self, idea):
                     return RatingSerializer(ratings.first()).data
         return None
 
+    def get_has_rating_permission(self, idea):
+        user = None
+        request = self.context.get('request')
+        if request and hasattr(request, 'user'):
+            user = request.user
+            return user.has_perm('a4_candy_ideas.rate_idea', idea)
+        return False
```

Comment:
> Just a second, a few more lines to the tests and the missing lines are also covered!

Well, even when we have no user logged in, the API gets the anonymous user, so I can't test these lines. But I would still like to keep them as is. So, test coverage is falling. :woman_shrugging: